### DISCUSSION TITLE
Marking False possitive

### DISCUSSION THREAD

**Discussion by mithu36** [kubescape/kubescape#1600] [General]
Hi, do kubescape have feature to mark some result as false possitive??
example:
having a text file where we can put the False positive test case and while kubescape doing the scan it will avoid hat one from the final result like IAC scanner?

**Comment by dwertent** [comment]
Hi, 
Sure, you can ignore failures by using [exceptions](https://kubescape.io/docs/accepting-risk/).

**Reply by mithu36** [reply]
Hi,
I tried this but when I'm ignoring failures with "controlID" the failure count of some other control names is increasing! 
why is it happening??
I tried scanning with exception and without exception that time i found this.

**Reply by dwertent** [reply]
Can you please attach an example so I can look into it?

**Reply by mithu36** [reply]
**screenshot after adding exception:**

<img width="762" alt="with-exception" src="https://github.com/kubescape/kubescape/assets/142890247/ffc2c308-9a0b-46f4-aeed-3fc088fb258c">

**without adding exception:**

<img width="749" alt="without-exception" src="https://github.com/kubescape/kubescape/assets/142890247/c55f8d32-b032-470f-920f-b4305e2b6abf">


as you can see in image the screenshot- after adding exception the failed number increases and other control name failed resource also increases

**Reply by dwertent** [reply]
This might happen because when you use your own exceptions file it overrides the default exceptions that we set up. The default exceptions typically exclude the kube-system related resources.

If you want to use your exceptions and the default exceptions, you should first download the default exceptions  `kubescape download `, add your exceptions to the downloaded file and use that when scanning.

**Reply by mithu36** [reply]
so i should run the command with modified exception??  
**kubescape scan --exceptions exceptions.json**
